### PR TITLE
Update Imaging Browser Readme

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
DICOM download has a specific php.ini setting for large files. Specify this in the README (identical description as in Dicom Archive)

### CODE REVIEW

File: modules/imaging_browser/README.md [hunk 1]
```diff
@@ -67,8 +67,11 @@ useProjects - This setting determines whether "project" filtering dropdowns exis
 useEDC - This setting determines whether "EDC" filtering dropdowns exist
         on the menu page.
 
-mantis_url - This setting defines a URL for LORIS to include a link to for bug reporting
-        on the "View Session" page.
+mantis_url - This setting defines a URL for LORIS to include a link to "mantis" for
```
Comment (driusan) [inline modules/imaging_browser/README.md]: issue_tracker_url

File: modules/imaging_browser/README.md [hunk 2]
```diff
@@ -67,8 +67,11 @@ useProjects - This setting determines whether "project" filtering dropdowns exis
 useEDC - This setting determines whether "EDC" filtering dropdowns exist
         on the menu page.
 
-mantis_url - This setting defines a URL for LORIS to include a link to for bug reporting
-        on the "View Session" page.
+mantis_url - This setting defines a URL for LORIS to include a link to "mantis" for 
+        bug reporting on the "View Session" page.
+
+For downloading large DICOM files, it may be necessary to increase the
+ value of the `memory_limit` configuration option within `php.ini`.
```
Comment (driusan) [inline modules/imaging_browser/README.md]: does this follow the same format as other READMEs for referring to php.ini configs?

File: modules/imaging_browser/README.md (hunk 1 above)
Comment (MounaSafiHarab) [reply modules/imaging_browser/README.md]: oh, did not know this changed!

File: modules/imaging_browser/README.md (hunk 2 above)
Comment (MounaSafiHarab) [reply modules/imaging_browser/README.md]: my initial commit had it as the same format as DICOM Archive. But I made both now similar to Imaging Uploader with two sub-sections, one related to install and one to database.

**Pull request merged**